core/aggregates/orgbilling/service.go

```diff
@@ -64,3 +68,65 @@ type AggregatedOrganization struct {
 func (s Service) Search(ctx context.Context, query *rql.Query) (OrgBilling, error) {
 	return s.repository.Search(ctx, query)
 }
+
+func (s Service) Export(ctx context.Context) ([]byte, string, error) {
+	orgBillingData, err := s.repository.Search(ctx, &rql.Query{})
+	if err != nil {
+		return nil, "", err
+	}
+	if err != nil {
+		return nil, "", err
+	}
+	// Create a buffer to write CSV data
+	var buf bytes.Buffer
+	writer := csv.NewWriter(&buf)
+
+	// Write CSV header
+	header := []string{
+		"Organization ID",
+		"Name",
+		"Title",
+		"Created By",
+		"Plan Name",
+		"Payment Mode",
+		"Country",
+		"State",
+		"Created At",
+		"Updated At",
+		"Subscription Cycle End",
+		"Subscription State",
+		"Plan Interval",
```

Comment:
Any way we can get this from struct field names itself?
This is basically hardcoding and seems like something can be missed in future.
It would be great if we can get it from json struct tags itself, even if the names do not have proper case.

Reply:
can you check now ?